DiscordChatExporter.Cli.dockerfile

```diff
@@ -1,22 +1,33 @@
 # Build
-FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:7.0-alpine AS build
+
+WORKDIR /build
 
 COPY favicon.ico ./
 COPY NuGet.config ./
 COPY Directory.Build.props ./
 COPY DiscordChatExporter.Core ./DiscordChatExporter.Core
 COPY DiscordChatExporter.Cli ./DiscordChatExporter.Cli
 
-RUN dotnet publish DiscordChatExporter.Cli --configuration Release --output ./publish
+RUN dotnet publish DiscordChatExporter.Cli \
+    --self-contained \
+    --use-current-runtime \
+    --configuration Release \
+    --output ./publish
 
 # Run
-FROM mcr.microsoft.com/dotnet/runtime:7.0 AS run
+FROM mcr.microsoft.com/dotnet/runtime-deps:7.0-alpine
 
-RUN useradd dce
+RUN apk add --no-cache tzdata && \
+    mkdir -p /opt/discord_chat_exporter && \
+    adduser \
+    --disabled-password \
+    --no-create-home \
+    dce
 USER dce
+COPY --from=build /build/publish /opt/discord_chat_exporter
 
-COPY --from=build ./publish ./
-
-WORKDIR ./out
+WORKDIR /out
```

Comment:
```suggestion
# Need to keep this as /out for backwards compatibility with documentation.
# A lot of people have this directory mounted in their scripts files, so
# changing it would break existing workflows.
WORKDIR /out
```